content/docs/user-guide/how-to/share-a-dvc-cache.md

```diff
@@ -0,0 +1,87 @@
+---
+title: 'How to Share a Cache Among Projects'
+description: >-
+  Setup a single cache for different projects or distributed copies of a same
+  project.
+---
+
+# How to Share a DVC Cache
+
+There are 2 main reasons to setup a shared <abbr>DVC cache</abbr>:
```

Comment:
both items are pretty hard to read to be honest

so, we have a few cases people would use this:

- one large machine (e.g. with multiple GPUs) and one storage on it and people do the same or multiple projects and we want to avoid duplication (no copies) and save time (no copy)
- one large NAS (<examples here>) and people attach it to their machines - again no copies - fast, doesn't take extra space, fits even if there are no space (ability to work with really large data)


what else am I missing? can we make description more explicit/simpler?

Reply:
Agreed. Updated, PTAL.